go-controller/pkg/ovn/controller/services/lb_config.go

```diff
@@ -832,3 +870,242 @@ func joinHostsPort(ips []string, port int32) []Addr {
 	}
 	return out
 }
+
+type lbEndpoints struct {
+	Port  int32
```

Comment:
Does it make sense for this port attribute to be here, or rather in the `lbConfig` struct?

Reply:
it's true that it's now duplicated in `clusterEndpoints` and `nodeEndpoints`. Let me see if I can move it up to `lbConfig`

Reply:
I was a bit afraid of taking this route. Can it wait?